api/RTCRtpContributingSource.json

```diff
@@ -234,10 +234,10 @@
               "version_added": false
             },
             "safari": {
-              "version_added": null
+              "version_added": "12.1"
```

Comment:
This (`RTCRtpContributingSource.timestamp`) was added in https://github.com/WebKit/WebKit/commit/e6d3663cfdb424aa1a684ad0d71169683f40f002 which was at trunk version 607.1.7:
https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/Configurations/Version.xcconfig?rev=236147

That maps to 12.1, so this checks out.